app/models/comment.rb

```diff
@@ -1,4 +1,13 @@
+# app/models/comment.rb
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_post_comments_counter
+
+  private
+
+  def update_post_comments_counter
+    post&.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] To meet the below requirement please add unit tests for this custom method created for the comment model :+1: 
![mm](https://github.com/NoelLincoln/blog-app/assets/89055502/1a7065fb-a00b-41cf-b3e5-3f72e0e2a85e)

Reply:
Thanks, I must have forgotten that bit.